tests/test_bin.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+
+from pacai.bin import pacman
```

Comment:
spacing and switch order

Reply:
I believe that Connor means to make sure that imports in the same group are sorted alphabetically.
(Not to swap lines 1 and 3.)

Reply:
I'm confused here, so is all I have to do is remove the line in between the imports? If I am not swapping lines 1 and 3.

Reply:
Imports should be in alphabetical order.
So remove the extra line and swap the order of the `pacai.bin` imports.